docker/storybook-server/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# syntax = docker/dockerfile:1.3
```

Comment:
Is docker never used for VRT in GitHub Actions?

Reply:
I remove it.